h2o-py/tests/testdir_jira/pyunit_pubdev_7237_effective_parameters.py

```diff
@@ -0,0 +1,268 @@
+import sys
```

Comment:
I think testdir_jira is more for is more for bugfixes, this is a new feature, my original suggestion was to split this into separate tests for each algo in testdir_algos, but I see that for your dev this is better because you want to test all the algos at once, but in the future you would usually change only one algo and want to touch only the test for that algo

Reply:
I also think we should add atleast a simple test for R